frontend/src/components/browse/BrowseContent.js

```diff
@@ -491,106 +495,108 @@ export default function BrowseContent({
         spinning: isFetchingMoreData || isFiltering,
       }}
     >
-      <Container maxWidth="lg">
-        <FilterSection
-          filtersExpanded={isMobileScreen ? filtersExandedOnMobile : filtersExpanded}
-          onSubmit={handleSearchSubmit}
-          setFiltersExpanded={isMobileScreen ? setFiltersExpandedOnMobile : setFiltersExpanded}
-          type={TYPES_BY_TAB_VALUE[tabValue]}
-          customSearchBarLabels={customSearchBarLabels}
-          filterButtonRef={filterButtonRef}
-          searchValue={filters.search}
-          hideFilterButton={tabValue === TYPES_BY_TAB_VALUE.indexOf("ideas")}
-        />
-        <Tabs
-          variant={isNarrowScreen ? "fullWidth" : "standard"}
-          value={tabValue}
-          onChange={handleTabChange}
-          indicatorColor="primary"
-          textColor="primary"
-          centered={true}
-        >
-          {TYPES_BY_TAB_VALUE.map((t, index) => {
-            const tabProps = {
-              label: type_names[t],
-              className: classes.tab,
-            };
-            if (index === TYPES_BY_TAB_VALUE.indexOf("ideas")) {
-              tabProps.label = (
-                <div className={classes.ideasTabLabel}>
-                  <EmojiObjectsIcon className={classes.ideasIcon} /> {type_names[t]}
-                </div>
-              );
-            }
-            if (index === 1) tabProps.ref = organizationsTabRef;
-            return <Tab {...tabProps} key={index} />;
-          })}
-        </Tabs>
-
-        <Divider className={classes.mainContentDivider} />
-
-        <>
-          <TabContentWrapper type={"projects"} {...tabContentWrapperProps}>
-            <ProjectPreviews
-              className={classes.itemsContainer}
-              hasMore={state.hasMore.projects}
-              loadFunc={() => handleLoadMoreData("projects")}
-              parentHandlesGridItems
-              projects={state.items.projects}
-              firstProjectCardRef={firstProjectCardRef}
-              hubUrl={hubUrl}
-            />
-          </TabContentWrapper>
-          <TabContentWrapper type={"organizations"} {...tabContentWrapperProps}>
-            <OrganizationPreviews
-              hasMore={state.hasMore.organizations}
-              loadFunc={() => handleLoadMoreData("organizations")}
-              organizations={state.items.organizations}
-              parentHandlesGridItems
-            />
-          </TabContentWrapper>
-          {!hideMembers && (
-            <TabContentWrapper type={"members"} {...tabContentWrapperProps}>
-              <ProfilePreviews
-                hasMore={state.hasMore.members}
-                loadFunc={() => handleLoadMoreData("members")}
+      <Suspense fallback={<LoadingSpinner isLoading={true} />}>
```

Comment:
Nice! I think this is our first usage of `<Suspense />` in the project. I personally haven't used it yet. Is it working well so far?

Reply:
Suspense is there because of the lazy loading. We fetch data asynchronously and we need some substitution while the content is being fetched. I am also wondering how much it will be faster in production :-)